curriculum/structure/superblocks/full-stack-developer.json

```
@@ -19,6 +19,7 @@
             "lecture-working-with-audio-and-video-elements",
             "lecture-working-with-images-and-svgs",
             "workshop-build-a-heart-icon",
+            "workshop-build-a-video-display-using-iframe",
             "lecture-working-with-media",
             "lab-video-compilation-page",
```

```suggestion
            "lecture-working-with-images-and-svgs",
            "workshop-build-a-heart-icon",
            "lecture-working-with-media",
            "workshop-build-a-video-display-using-iframe",
            "lab-video-compilation-page",
```
following the spreadsheet, this seems to be the right place